chips/cc26x2/src/chip.rs

```diff
@@ -13,12 +13,12 @@ pub struct Cc26X2 {
 }
 
 impl Cc26X2 {
-    pub unsafe fn new() -> Cc26X2 {
+    pub unsafe fn new(hfreq: u32) -> Cc26X2 {
```

Comment:
Can you document what `hfreq` is supposed to be and where this number comes from?

Is the clock frequency a parameter that the board gets to choose? Or is it property of the particular variant of the chip (and if so, where should the board integrator look in the datasheet)?

Is it a discrete set of possible options (in which case it should probably be an enum), or can it be any number within some range (in which case the range should be specified)?

What are the units? (probably Hz)

Reply:
Comment added. Thanks for the note